### PR TITLE
Release blog post for AeroGear WebPush.

### PR DESCRIPTION
**Pull request by danbev**
Note:
This file should be renamed to an appropriate date. It is only named to
an earlier data to show up for those that need to review it.

### CODE REVIEW

File: _posts/2015-02-17-aerogear-webpush.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+layout: post
+title: AeroGear WebPush
+---
+
+## AeroGear WebPush
+[AeroGear WebPush](https://github.com/aerogear/aerogear-webpush-server#aerogear-webpush-server-) is a proof of concept implementation of the 
+[WebPush Protocol](http://tools.ietf.org/html/draft-thomson-webpush-http2-02) specification with the purpose of gaining a better understanding of the specification. 
+
+As you might have guessed from the name this is about push notifications and is related to the [Push API](https://w3c.github.io/push-api/index.html). 
+In brief, it allows a device to maintain a single HTTP/2 connect which can service as many client applications as needed. For example, when used in combination with the Push API a service worker will handle the registration to the WebPush Server and manage the subscriptions for the applications. An application would in this case be tab in a web browser.
```
Comment (matzew) [inline _posts/2015-02-17-aerogear-webpush.md]: for _service worker_, should we link to ? http://www.html5rocks.com/en/tutorials/service-worker/introduction/

File: _posts/2015-02-17-aerogear-webpush.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: AeroGear WebPush
+---
+
+## AeroGear WebPush
```
Comment (matzew) [inline _posts/2015-02-17-aerogear-webpush.md]: let's be specific that this is an aplha.1 release - I think we should rename the file to reflect that as well

File: _posts/2015-02-17-aerogear-webpush.md (hunk 1 above)
Comment (danbev) [inline _posts/2015-02-17-aerogear-webpush.md]: done

File: _posts/2015-02-17-aerogear-webpush.md (hunk 2 above)
Comment (danbev) [inline _posts/2015-02-17-aerogear-webpush.md]: done

**Comment by matzew** [comment]
@danbev looks good to me! Do you know when the netty 4.1.0.beta4 will be released ?

**Comment by danbev** [comment]
@matzew Should be this week.

File: _posts/2015-02-17-aerogear-webpush-alpha1.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: AeroGear WebPush
```
Comment (matzew) [inline _posts/2015-02-17-aerogear-webpush-alpha1.md]: the value of the title is used on the "Hey listen! ...." section of the side.

I'd suggest to use a different title, like -> _AeroGear WebPush alpha.1 is out_.

The generated content on the site would be "Hey listen! AeroGear WebPush alpha.1 is out"

Comment (danbev) [inline _posts/2015-02-17-aerogear-webpush-alpha1.md]: A right, I've update it now and it looks right to me. @matzew Can you double check?

**Comment by matzew** [comment]
@danbev yes, good

**Comment by matzew** [comment]
Should we close the PR until we really have the release out? We did similar in the past other repos, to avoid accidental merges ...

**Comment by danbev** [comment]
@matzew Sure nps.

**Pull request closed without merging**